doc/common/breaking_changes.md

```diff
@@ -0,0 +1,104 @@
+This document contains a list of known breaking changes in the engineering system tools (mainly `eng/common` directories) and available workarounds. Typically this takes the form of infrastructure changes that are not backwards compatible with older versions of the engineering system configs/scripts/tools, mainly due to dependency changes in our test agent images.
+
+Table of Contents
+=================
+
+* [Credscan - You must install or update .NET to run this application.](#credscan---you-must-install-or-update-net-to-run-this-application)
+    * [Workaround](#workaround)
+* [fatal: specify directories rather than patterns](#fatal-specify-directories-rather-than-patterns)
+    * [Workaround](#workaround-1)
+
+
+## Credscan - You must install or update .NET to run this application.
+
+This error comes from the "CredScan running" step in the Compliance stage:
```

Comment:
> in the Compliance stage

This is a bit vague to me because I am missing context. Compliance stage of what? Some build? Of which pipeline? An example would be usfeful.

Reply:
Good point, in this one it's pretty consistent which azure pipeline step this comes from. I will link to the config and add a screenshot.